processes/org-monthly-meetings/minutes/2019-06-Minutes.md

```diff
@@ -0,0 +1,173 @@
+|Handbook for organizers|@amaliacardenas|2|
+|Raquel and Salva on-boarding|@jmbriano|3|
+|Instagram account|@raquelalvear|5|
+|Cloning RGSoC|@amaliacardenas|2|
+|Hosting events at King|@amaliacardenas|5|
+
+
+## New initiatives (20')
+
+|**Initiative**| Presented by |minutes|
+|:------------|--------------|:--:|
+|Adding a community manager|@loriking| 5|
+
+
+# Ongoing initiatives
+
+## Specific workshops/Curriculum development
+
+Salva and Lori discussed ideas to develop careers and learning paths. They identified that people who attend the codebar workshops are often confused regarding the different types of career paths that they can take. As part of the workshop series that we can organize, Lori started to discuss with somebody from the Thoughtworks NY office, who is part of the Thoughtworks Arts program and is interested in giving a workshop on the ways to collaborate with artists. After having this workshop,  we propose to have follow sessions during the regular codebar workshops so that students can work directly on what was discussed.
+
+A second workshop could be one that highlights the diversity of jobs available in tech.
+
+One important thing that we are trying to address is once students learn the basics, where can they go next? We need to find out what students need.
+
+We also discussed the possibility of forming codebar teams. Were a team of people work on a project together. We discussed creating a pilot group of 5 people to see how we could scale this up to the large codebar community.
+
+The idea would be to start these workshops in September.
+
+**Owner:** @loriking
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Define process for selecting next hosts
+The host selection sheet has been completed. A section to add historical data needs to be included and all Barcelona team members need to complete the host rating. There is a possibility of adding more dates to the calendar, but this will not happen until September. There is a possibility of having a session in August, although it might be a more social session.
+
+**Owner:** @amaliacardenas
+
+**Status:**
+This initiative is **ongoing** and should be included in next month agenda.
+
+
+## Codebar landing page. Explanation in 3 languages
+David attended the monthly organizer's meeting and discussed why it is important for the Barcelona chapter to have an explanation in three languages. Some of our students do not speak English, and we need a way for them to feel welcome and safe during the workshops. There is a possibility to translate the entire website. The translation project can be turned into a workshop. It will be a good way to include our students and it's a great way to learn git. There is a UX designer from London who is working on redesigning the landing page so that the page can be in different languages. It will take some time for the redesign to be completed, but the project is progressing.  
```

Comment:
```suggestion
David attended the monthly organizer's meeting and discussed why it is important for the Barcelona chapter to have an explanation in three languages. Some of our students do not speak English, and we need a way for them to feel welcome and safe during the workshops. There is a possibility to translate the entire website. The translation project can be turned into a workshop. It will be a good way to include our students and it's a great way to learn git. There is a UX designer from Oslo(?¿) who is working on redesigning the landing page so that the page can be in different languages. It will take some time for the redesign to be completed, but the project is progressing. To follow the progress: https://github.com/codebar/chapter-organiser-meetings/issues/17
```

Reply:
the ux designer is not from London and I've added the link to the issues.

Reply:
@amaliacardenas could you commit this suggestion? (it is on your fork repo, I cannot)